src/current/v23.1/sso-db-console.md

```diff
@@ -6,94 +6,104 @@ docs_area: manage
 
-CockroachDB supports SSO via [OpenID Connect (OIDC)](https://openid.net/connect/), an authentication layer built on top of OAuth 2.0.
+This page describes:
+
+- The login flow for users, once a cluster is SSO-enabled.
+- Configuring a CockroachDB Cluster for OIDC SSO authentication to the DB Console.
+- Provioning an OIDC client for CockroachDB Cluster SSO authentication, using GCP.
+
+Additional configuration is needed to enable SSO authentication for SQL clients, which is described here: [Cluster Single Sign-on (SSO) using a JSON web token (JWT)]({% link {{ page.version.version }}/sso-sql.md %}) 
 
 {{site.data.alerts.callout_info}}
-SSO for DB Console is available to CockroachDB {{ site.data.products.cloud }} customers.
+SSO for DB Console is available to CockroachDB {{ site.data.products.dedicated }} customers.
 
 On CockroachDB {{ site.data.products.core }} clusters, SSO for DB Console requires an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
 {{site.data.alerts.end}}
 
-## Requirements
+This SSO implementation uses the [authorization code grant type](https://tools.ietf.org/html/rfc6749#section-4.1) to authenticate a user.
 
-- An external OAuth 2.0 identity provider
+**Prerequisites**:
 
-## Login flow
+- **IdP:** You must have access to an OAuth 2.0 identity provider and client. The process of provisioning one with Google Cloud Platform is described here: [Provisioning an OAuth client using Google Cloud Platform (GCP)](#provision-an-oauth-client-using-google-cloud-platform-gcp)
 
-This SSO implementation uses the [authorization code grant type](https://tools.ietf.org/html/rfc6749#section-4.1) to authenticate a user.
+- **CockroachDB Cluster**: you must have access to one of the following:
+	1. A {{ site.data.products.core }} cluster enabled with a valid [CockroachDB Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
+	1. A [CockroachDB {{ site.data.products.dedicated }}cluster](https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster).
+
+
+## Log in to a cluster's DB Console with SSO
 
-1. A user opens the DB Console and clicks on the OAuth login button.
+From the user's perspective, once the cluster is properly configured to an identity provider, the sign-in flow is as follows:
+
+1. A user opens the cluster's DB Console, and clicks on the OAuth login button which renders.
 1. The user is redirected to an external identity provider.
 1. The user successfully authenticates with the provider, completing the OAuth flow.
-1. The user is redirected to the CockroachDB cluster via a callback URL.
+1. The user is redirected to the CockroachDB cluster.
 1. The authorization code in the callback query is exchanged for an [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
```

Comment:
Nit without the callback reference in the line above it may not be immediately clear what callback query it is referring to. It might also be worth calling it "in the callback query parameter"

Reply:
revised this a little. i think all that about the callback query is TMI since it's invisible to the user just trying to log in in the browser.

Reply:
👍